lib/docker.js

```diff
@@ -464,16 +464,17 @@ Docker.prototype.getEvents = function(opts, callback) {
  * @param  {String}   repoTag  Repository tag
  * @param  {Object}   opts     Options (optional)
  * @param  {Function} callback Callback
+ * @param  {Object}   auth     Authentication (optional)
  * @return {Object}            Image
  */
-Docker.prototype.pull = function(repoTag, opts, callback) {
+Docker.prototype.pull = function(repoTag, opts, callback, auth) {
```

Comment:
Of course it would, but then it would become a breaking change. Since I have no legacy code depending on Dockerode, this would suit fine, but I cannot speak for others.